addons/pkg/util/clusterbootstrapclone/clusterbootstrapclone.go

```diff
@@ -368,6 +368,8 @@ func (h *Helper) HandleExistingClusterBootstrap(clusterBootstrap *runtanzuv1alph
 					}
 				}
 			}
+			h.Logger.Info("Updating packagesToBeCloned with additional packages")
+			packagesToBeCloned = append(packagesToBeCloned, clusterBootstrap.Spec.AdditionalPackages...)
```

Comment:
I think the right way is to add the following block to line 344 https://github.com/vmware-tanzu/tanzu-framework/blob/main/addons/pkg/util/clusterbootstrapclone/clusterbootstrapclone.go#L344

```
var nonNilValuesFromPackages []*runtanzuv1alpha3.ClusterBootstrapPackage
			for _, cbPackage := range packages {
				if cbPackage != nil && cbPackage.ValuesFrom != nil {
					nonNilValuesFromPackages = append(nonNilValuesFromPackages, cbPackage)
				}
			}
```

And the do the following instead in line 371 above:
```
packagesToBeCloned = append(packagesToBeCloned, nonNilValuesFromPackages...)
```

Reply:
used the suggested fix